src/assets/styles/global/colors.scss

```diff
@@ -3,3 +3,6 @@ $p-error-color: #ff0000;
 $p-color-grey: #c3c3c3;
 $p-color-white: #ffffff;
 $p-color-black: #000000;
+$p-color-17: #171717;
```

Comment:
Can we relate 17 to something more descriptive? as red light red dark, or maybe more to context input border etc...

Reply:
We should talk about it. For now, let it stay as it is here.